addons/mail/static/src/messaging.js

```diff
@@ -36,6 +37,7 @@ export class Messaging {
         this.threads = {};
         this.users = {};
         this.internalUserGroupId = null;
+        this.registeredImStatusPartners = markRaw([]);
```

Comment:
Why you need `markRaw` here?
If a component observes it, then should be reactive too.

Reply:
I needed a list not proxy to pass to registerToImStatus function. I moved markRaw to  updateImStatusRegistration function.